Runtime/Interaction/SnapZone.cs

```diff
@@ -131,19 +130,20 @@ public Material InvalidMaterial
         /// </summary>
         protected XRBaseInteractable ForceSelectTarget { get; set; }
         
-        private Mesh previewMesh;
+        private Mesh previewMeshes;
         
-        internal Mesh PreviewMesh 
+        internal Mesh PreviewMeshes 
```

Comment:
Not sure if I like this internal tbh. Not being able to get access to members is one of the most annoying things in Unity XR

Reply:
Why would you want to access this member?